_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -55,6 +55,8 @@ Some browsers restrict scrolling to the [content box](https://drafts.csswg.org/c
 
 To ensure there is some element from which arrow keys can be used to control the scroll position, focus must be on or in a scrollable region. If scripts are used to prevent the keyboard events from reaching the scrollable region, this could still cause a keyboard accessibility issue. This must be tested separately.
 
+This rule is only applicable to non-embedded scrollable elements. Embedded scrollable elements, such as iframe or object elements, are covered in other rules such as [Iframe with negative tabindex has no interactive elements](iframe-not-focusable-has-no-interactive-content-akn7bn.md).
```

Comment:
```suggestion
This rule only applies to elements scroll content in the same document. Elements such as iframes that embed other documents may also show be scrollable, but for them it is the embedded document that scrolls, not the content in the same document. Such scenarios are tested separately wit rules such as [Iframe with negative tabindex has no interactive elements](https://act-rules.github.io/rules/akn7bn).
```

Reply:
I am good with the changes.  Spelling correction:  "...separately wit**h** rules such as [**i**Frame with negative tabindex has no interactive elements]...

Reply:
```suggestion
This rule only applies to elements who scroll content in the same document. Elements such as iframes that embed other documents may also be scrollable, but for them it is the embedded document that scrolls, not the content in the same document. Such scenarios are tested separately with rules such as [Iframe with negative tabindex has no interactive elements](https://act-rules.github.io/rules/akn7bn).
```
(correcting some typos on @WilcoFiers' suggestion)